neurodamus/ngv.py

```diff
@@ -418,7 +419,8 @@ def finalize(self, base_Seed=0, *_):
                 "Missing cells to couple Glia to: %d", len(NeuroGlialConnection.neurons_not_found)
             )
 
-    def _create_synapse_ustate_endpoints(self, base_manager):
+    @staticmethod
```

Comment:
the same as before. Once we agree on the point for one, you can close all of these